_data/devops-checklist.yml

```diff
@@ -0,0 +1,690 @@
+        to use a <a href="https://aws.amazon.com/elasticloadbalancing/" target="_blank">Load Balancer</a> for health
+        checks and to automatically replace failed EC2 Instances. Use your Docker orchestration tool to monitor the
+        health of your Docker containers and automatically restart failed ones (e.g.,
+        <a href="https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_HealthCheck.html" target="_blank">ECS Health Checks</a>).
+
+    - title: Configure graceful degradation
+      description: |
+        Handle failures in your dependencies (e.g., a service not responding) by using
+        <a href="https://twitter.com/copyconstruct/status/994138694582812672" target="_blank">graceful degradation patterns</a>,
+        such as retries (with exponential backoff and jitter), circuit breaking, timeouts, deadlines, and rate limiting.
+
+    - title: Perform load tests and use chaos engineering
+      description: |
+        Run load tests against your infrastructure to figure out when it falls over and what the bottlenecks are. Use
+        <a href="https://principlesofchaos.org/" target="_blank">chaos engineering</a> to continuously test the
+        resilience of your infrastructure (see also
+        <a href="https://github.com/Netflix/chaosmonkey" target="_blank">chaos monkey</a>).
+
+- category: Continuous Integration
+  items:
+    - title: Pick a Version Control System
+      description: |
+        Check all code into a Version Control System (VCS). The most popular choice these days is
+        <a href="https://git-scm.com/" target="_blank">Git</a>. You can use
+        <a href="https://github.com/" target="_blank">GitHub</a>, <a href="https://gitlab.com/" target="_blank">GitLab</a>,
+        or <a href="https://bitbucket.org/" target="_blank">BitBucket</a> to host your Git repo.
+
+    - title: Do code reviews
+      description: |
+        Set up a code review process in your team to ensure all commits are reviewed.
+        <a href="https://help.github.com/articles/about-pull-requests/" target="_blank">Pull requests</a> are an easy
+        way to do this.
+
+    - title: Configure a build system
+      description: |
+        Set up a build system for your project, such as <a href="https://gradle.org/" target="_blank">Gradle</a> (for
+        Java), <a href="https://github.com/ruby/rake" target="_blank">Rake</a> (for Ruby), or
+        <a href="https://yarnpkg.com/en/" target="_blank">Yarn</a> (for Node.js). The build system is responsible for
+        compiling your app, as well as many other tasks described below.
+
+    - title: Use dependency management
+      description: |
+        Your build systems should allow you to explicitly define all the of the dependencies for your apps. Each
+        dependency should be versioned, and ideally, the versions of all dependencies, including transitive
+        dependencies, are captured in a lock file.
```

Comment:
May need some more detail around the "lock" file or a link where you can read more about it.

Reply:
Good point. Added a couple links. https://github.com/gruntwork-io/gruntwork-io.github.io/pull/45/commits/ac969266fa45512affe5434f70111b488f8c4eb8